forge-gui/res/cardsfolder/upcoming/omnath_locus_of_all.txt

```diff
@@ -0,0 +1,14 @@
+Name:Omnath, Locus of All
+ManaCost:W U P R G
+Types:Legendary Creature Phyrexian Elemental
+PT:4/4
+R:Event$ LoseMana | ValidPlayer$ You | ReplacementResult$ Replaced | ReplaceWith$ ConvertMana | ActiveZones$ Battlefield | Description$ If you would lose unspent mana, that mana becomes black instead.
+SVar:ConvertMana:DB$ ReplaceMana | ReplaceType$ B
```

Comment:
Problem: Omnath uses ColorWord, that means it should be affected by CardTextChanges, that's why it needs to be "Black"